odxtools/diagdatadictionaryspec.py

```diff
@@ -29,37 +30,55 @@
 
 @dataclass
 class DiagDataDictionarySpec:
+    admin_data: Optional[AdminData]
```

Comment:
is there another element that should have admin data but does not have it yet?

Reply:
that's possible: I don't remember going through the XSD and checking all occurences (and even if I did, I clearly missed this occasion ;)). That said, I'm pretty sure that we've caught the vast majority of them...

anyway, `DiagDataDictionarySpec` should now internalize all subtags of the corresponding XML element (modulo DYNAMIC-ENDMARKER-FIELDS)